### DISCUSSION TITLE
How can I disable transition temporarily ?

### DISCUSSION THREAD

**Discussion by cottonspace** [fengyuanchen/viewerjs#568] [Q&A] [answered]
Hello. Thanks for creating such a cool application.

I want to *rotateTo* and *scale* at **initial display**.

```
viewed() {
    this.viewer.rotateTo(initialDegree).scale(initialScaleX, initialScaleY)
}
```

However, **no** transition effect is needed at that time. How can I turn off transition effects for initial display as temporarily?

Regards.

**Comment by fengyuanchen** [comment, marked as answer]
Not support that currently.

**Reply by cottonspace** [reply]
Thank you for your answer.

Regards,
